internal/services/manageddevopspools/managed_devops_pool_data_source_test.go

```diff
@@ -0,0 +1,42 @@
+package manageddevopspools_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/location"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+)
+
+type ManagedDevOpsPoolDataSource struct{}
+
+func TestAccManagedDevOpsPoolDataSource_basic(t *testing.T) {
+	if os.Getenv("ARM_MANAGED_DEVOPS_ORG_URL") == "" {
+		t.Skip("Skipping as `ARM_MANAGED_DEVOPS_ORG_URL` is not specified")
+	}
+
+	data := acceptance.BuildTestData(t, "data.azurerm_managed_devops_pool", "test")
+	r := ManagedDevOpsPoolDataSource{}
+
+	data.DataSourceTest(t, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).Key("location").HasValue(location.Normalize(data.Locations.Primary)),
```

Comment:
Can we add more checks on those computed attributes to ensure they are correctly set during `Read()`?